agent/acs/handler/acs_handler.go

```diff
@@ -325,9 +312,13 @@ func (acsSession *session) startACSSession(client wsclient.ClientServer) error {
 	responseSender := func(response interface{}) error {
 		return client.MakeRequest(response)
 	}
-
-	heartbeatResponder := acssession.NewHeartbeatResponder(acsSession.doctor, responseSender)
-	client.AddRequestHandler(heartbeatResponder.HandlerFunc())
+	responders := []wsclient.RequestResponder{
+		acssession.NewAttachTaskENIResponder(eniHandler, responseSender),
```

Comment:
don't see a `start()` and `defer stop()` here. What's changed?

Reply:
There is no need to do multi-thread synchronization with channels with this responder. As part of the refactor, we factor out this non-necessity and thus `start()` and `stop()` functions can be removed. The resulting responder is a lot cleaner and simpler to follow.

See the following PR for a similar change previously made to the heartbeat responder for the same reason:
https://github.com/aws/amazon-ecs-agent/pull/3232

Reply:
Thanks!